neutrons/SNAPRed
# Autogenerated api docs

**Description of work**
Generated documentation of code using sphinx. Modified the conf.py file to get `make html` working correctly.
**To test:**

While in the docs folder, run `make html` to see what the new read.the.docs pages will look like.

Link to EWM item
[Task 2399](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=2399)
